internal/storage/ssblob/ssblob_test.go

```diff
@@ -0,0 +1,46 @@
+package ssblob_test
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"gotest.tools/v3/assert"
+
+	"github.com/artefactual-sdps/enduro/internal/storage/ssblob"
+)
+
+func TestBucket(t *testing.T) {
+	t.Parallel()
+	handler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "image/jpeg")
+		w.Write([]byte("hello"))
+	})
```

Comment:
This is where we can pretend we're the Storage Service. Here you can look up the request (r) and confirm the headers (e.g. Authorization header with the API key), URL or HTTP method. You can assert with gotest.tools/v3/assert here too.